front/src/components/CardDetails.vue

```diff
@@ -124,8 +128,24 @@ const goback = (bool: boolean) => {
 
       <v-chip>Chip 3</v-chip>
     </v-chip-group>
-    <v-btn block size="large" color="green"><strong>+</strong> Ajouter à l'intinéraire </v-btn>
-    <v-btn block size="large" color="red"><strong>-</strong> Retirer à l'intinéraire </v-btn>
+    <v-btn
+      block
+      size="large"
+      color="green"
+      prepend-icon="mdi-plus"
+      v-if="!mapPointsStore.includes($props.data)"
+      @click="mapPointsStore.add($props.data)"
+      >Ajouter à l'intinéraire</v-btn
+    >
+    <v-btn
+      block
+      size="large"
+      color="red"
+      prepend-icon="mdi-minus"
+      v-else
+      @click="mapPointsStore.remove($props.data)"
+      >Retirer de l'intinéraire</v-btn
+    >
```

Comment:
You use the same button on MainLayout.vue

Better to create a unique component ItinaryButton
He can take props like: 
-action for the @click
-contentText (Add or remove)
- other css props